recipes/soil2/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment, MSBuild
+import os
+
+
+class soil2Conan(ConanFile):
+    name = "soil2"
+    description = "Simple OpenGL Image Library 2"
+    topics = ("conan", "soil2", "opengl", "images")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/SpartanJ/SOIL2"
+    license = "MIT-0"
+    settings = "os", "arch", "compiler", "build_type"
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+    generators = "premake"
+
+    def config_options(self):
+        # Visual Studio users: SOIL2 will need to be compiled as C++ source ( at least the file etc1_utils.c ), since VC compiler doesn't support C99
+        del self.settings.compiler.libcxx
```

Comment:
The libcxx is there for the c++ runtime library.
gcc has libstdc++ and libstdc++11.
clang has libstdc++, libstdc++11 and clang++.
MSVC has no specific c++ runtime, only its general runtime (MT/MD/MTd/MDd)